tests/integration/auth-error.test.js

```diff
@@ -100,9 +164,10 @@ const test = async app => {
         object: {
           type: 'Note',
           content: 'This is the content of note A.',
-          'oa:hasSelector': {},
-          context: publicationUrl,
-          inReplyTo: documentUrl
+          selector: { property: 'value' },
+          json: { anotherProperty: 3 },
+          documentId: idToUrl(document.id, 'document'),
+          publicationId: resPublication.body.id
```

Comment:
look at the note.test file to see how to create a note with a context and inReplyTo